coops/Titans_stronghold.tex

```diff
@@ -0,0 +1,201 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Cooperative scenario}{Titans' stronghold}{\images/Stronghold.png}
+
+\begin{multicols*}{2}
+
+\textbf{Author:} Invoceusse
+
+\textbf{Source:} \href{https://discord.com/channels/740870068178649108/1219333721019256943}{Archon Studio Discord}
+
+\subsection*{\MakeUppercase{The story}}
+
+A long time ago, a powerful fortress was built to house the fantastic and murderous Titans. No one has ever entered and returned to tell the tale. Now, peoples speak about death of guardians and an incredibly lot of treasures, with potentially the best bow of the world. 
+
+If this legend of the empty fort is true, it's the time for excavation.
+
+But for now, you and your friends need to find the keys spread around Antagarich to open the gate of Titans' stronghold. 
+
+\subsection*{\MakeUppercase{Scenario length}}
+
+This scenario plays out over 16 rounds. (15 in impossible)
+
+\subsection*{\MakeUppercase{Player setup}}
+
+\textbf{Player count:} 1 - 6
+
+\textbf{Starting Resources:} 30 \svg{gold}, 8 \svg{building_materials}, 2 \svg{valuables}
+
+\textbf{Starting Income:} 10 \svg{gold}, 2 \svg{building_materials}, 1 \svg{valuables}
+
+\textbf{Starting Units:} 
+    A Pack of \svg{bronze} units of your choice.
+    A Few \svg{bronze} units of your choice.
+
+\textbf{Town Buildings:} nothing
+
+\subsection*{\MakeUppercase{Map setup}}
+
+Take the following Map tiles and set them up as shown in the scenario map layout: (P for number of player)
+
+\textbf{(1*P) Starting Map tile (I)}
+\begin{itemize}
+    \item Starting Tiles of your chosen factions.
+    \item Ignore all of their yellow line.
```

Comment:
```suggestion
    \item Ignore all of their yellow borders.
```

Reply:
Even for this line? 🤔 
![image](https://github.com/qwrtln/Homm3BG-mission-book/assets/163738667/1708908e-d95a-4e09-bcda-b3d468e8f525)

Reply:
Well, isn't it what you meant?

Reply:
I think the original defines it as a border
![image](https://github.com/qwrtln/Homm3BG-mission-book/assets/5611195/2bd8127b-a78c-490b-825b-0c46cb0dc7ff)

Reply:
![image](https://github.com/qwrtln/Homm3BG-mission-book/assets/5611195/20a6cd57-8e7a-459a-ba0f-3ec8ca074997)